lib/authorize.js

```diff
@@ -24,16 +24,16 @@ export default async function authorize (insecure) {
 
     if (!process.env.HOME) {
       throw new Error('Could not determine your $HOME');
-    } 
-    
+    }
+
     user = /\/([^\/]+)$/.exec(process.env.HOME)[1];
-    xcodeDir = await getPath(); 
-    logger.warn(`The xcode directory is : ${xcodeDir}`);
-  } catch (e) {  
+    xcodeDir = await getPath();
+    logger.info(`The xcode directory is : ${xcodeDir}`);
+  } catch (e) {
     logger.errorAndThrow(e);
   }
 
-  try { 
+  try {
     // change permission
```

Comment:
sure. They're this way because we need to cover ourselves for whichever version of Xcode the user has installed, right? Older versions of xcode use the first one?